### ISSUE TITLE
Add 'Other' Content Category

### ISSUE THREAD

**Issue by MichaelGHUser** [bluesky-social/social-app#1195]
**Is your feature request related to a problem? Please describe.**

Available content filters are broad.

**Describe the solution you'd like**

Having this option available in Moderation settings and individual posts would give users the ability to be a bit more specific with their warnings (similar to Mastodon).

**Additional context**

News moves fast, and talking about difficult subjects is sometimes necessary.  In example:

```
Content warning: Other*
politics, climate change, natural disasters, disturbing content/images, Maui wildfires
``` 

_*User-defined terms (abuse of this feature can result in temporary or permanent suspensions)_

Unorthodoxed uses: 
 
Shortly after joining Mastodon, a disabled user asked me to use the content warning box when posting my Wordle results.  By putting the gameboard results in the body of the message and the results in the content warning box, it made my Wordle posts more compatible with screen readers.

**Comment by MichaelGHUser** [comment]
Arguably, a user could mute certain terms, but that seems tedious to mute and unmute.  Muting terms, at least for me, is a more permanent thing, whereas someone's spoons could change significantly throughout the day.

**Comment by mschwendener** [comment]
If it can work as on Mastodon (= the characters you use for the visible text of the 'Other' label are distracted from the total character count of your post), it's more like a page break or 'above the fold / below the fold'. 

Muting words happens on the reader's side, setting a 'page break' is a tool for the writer.

**Comment by MichaelGHUser** [comment]
> If it can work as on Mastodon (= the characters you use for the visible text of the 'Other' label are subtracted from the total character count of your post), it's more like a page break or 'above the fold / below the fold'.
> 
> Muting words happens on the reader's side, setting a 'page break' is a tool for the writer.
> 
> _Edit:_ seeing in an [article](https://maloki.medium.com/on-content-warnings-on-mastodon-318771be78e8) that what I mean are Content Notes (paragraph 2, can't see the whole article)

Yeah, you have to be a member apparently.  I remember when Medium was free. 😟

**Comment by pfrazee** [comment]
Yeah I've been planning on this -- there is a general label `!warning` which applies to everyone (regardless of their content filtering settings) but which only covers the post (no filtering). If we add an open description field, it could show more info in the cover UI. See https://github.com/bluesky-social/atproto/discussions/1466#discussioncomment-6722844

**Comment by MichaelGHUser** [comment]
I did not realize this exists.  How does the syntax work?  Something like:

`!warning Disturbing content` followed by the post body?

**Comment by mschwendener** [comment]
> If we add an open description field

_Description_ is a good way to put it, yes. 

Use the labels for Content warning (as it is now), and introduce a description field that can but doesn't have to be used in combination with a label. 

Label is stronger than description: when there is only a description, without a label, the description is shown without a click. 

When there is a label, I do not want to see the description without clicking.

**Comment by MichaelGHUser** [comment]
> Yeah I've been planning on this -- there is a general label `!warning` which applies to everyone (regardless of their content filtering settings) but which only covers the post (no filtering). If we add an open description field, it could show more info in the cover UI. See [bluesky-social/atproto#1466 (reply in thread)](https://github.com/bluesky-social/atproto/discussions/1466#discussioncomment-6722844)

This `!warning` label did not appear to work when I made a test post, so it's possible I am doing something incorrectly.

**Comment by mschwendener** [comment]
https://bsky.app/profile/bsky.app/post/3k4z36jmspl2g 

"We’ve started with the NSFW categories, but will expand them over time."

Planned list here https://github.com/bluesky-social/atproto/blob/73f64009a22b3428a21a6d7807aee89d629fd0a4/packages/api/docs/labels.md 

"still in development and may change over time. Not all are currently in use."

**Comment by noritada** [comment]
I would like to put +1 to this feature proposal and look forward to seeing it introduced.

In the first two days of the new year, Japan has had a major earthquake and a plane crash, requiring both time to talk about the sad news and time to distance oneself from the news to prevent compassion fatigue caused by all the sad news.
The same thing can happen in other countries, so we hope to see the introduction of "other" content warnings.
